### ISSUE TITLE
Check failed: error == cudaSuccess (17 vs. 0) invalid device pointer

### ISSUE THREAD

**Issue by ergysr** [chuckcho/video-caffe#25]
Hi,

I am trying to run the UCF example but this error happens immediately after prefetching data completes. The MNIST example however works fine. Do you have any suggestions how to solve it? I run Ubuntu 16.04, Cuda Toolkit 7.5 and cuDNN v5. 

> 
> I0628 20:08:20.031828 18205 video_data_layer.cpp:204] Prefetch batch: 322 ms.
> I0628 20:08:20.031852 18205 video_data_layer.cpp:205]      Read time: 77.958 ms.
> I0628 20:08:20.031857 18205 video_data_layer.cpp:206] Transform time: 233.47 ms.
> I0628 20:08:20.330770 18205 video_data_layer.cpp:204] Prefetch batch: 297 ms.
> I0628 20:08:20.330793 18205 video_data_layer.cpp:205]      Read time: 54.76 ms.
> I0628 20:08:20.330797 18205 video_data_layer.cpp:206] Transform time: 226.509 ms.
> I0628 20:08:20.634552 18205 video_data_layer.cpp:204] Prefetch batch: 302 ms.
> I0628 20:08:20.634577 18205 video_data_layer.cpp:205]      Read time: 58.31 ms.
> I0628 20:08:20.634582 18205 video_data_layer.cpp:206] Transform time: 233.66 ms.
> I0628 20:08:20.833590 17879 solver.cpp:404]     Test net output #0: accuracy/top-1 = 0.004
> I0628 20:08:20.833631 17879 solver.cpp:404]     Test net output #1: accuracy/top-5 = 0.042
> I0628 20:08:20.833642 17879 solver.cpp:404]     Test net output #2: loss = 4.74786 (* 1 = 4.74786 loss)
> F0628 20:08:21.043576 17879 dropout_layer.cu:31] Check failed: error == cudaSuccess (17 vs. 0)  invalid device pointer
> *** Check failure stack trace: ***
>     @     0x7fda4b8aa5cd  google::LogMessage::Fail()
>     @     0x7fda4b8ac433  google::LogMessage::SendToLog()
>     @     0x7fda4b8aa15b  google::LogMessage::Flush()
>     @     0x7fda4b8ace1e  google::LogMessageFatal::~LogMessageFatal()
>     @     0x7fda4c283abc  caffe::DropoutLayer<>::Forward_gpu()
>     @           0x421948  caffe::Layer<>::Forward()
>     @     0x7fda4c240eea  caffe::Net<>::ForwardFromTo()
>     @     0x7fda4c240baf  caffe::Net<>::Forward()
>     @     0x7fda4c0cb004  caffe::Net<>::ForwardBackward()
>     @     0x7fda4c0c358a  caffe::Solver<>::Step()
>     @     0x7fda4c0c2f19  caffe::Solver<>::Solve()
>     @           0x41c2a2  train()
>     @           0x41e5a1  main
>     @     0x7fda4a360830  __libc_start_main
>     @           0x41adb9  _start
>     @              (nil)  (unknown)
> Aborted (core dumped)

Thanks!

**Comment by ergysr** [comment]
Found the answer in this pull request: https://github.com/chuckcho/video-caffe/pull/24
